docs/BR.md

```diff
@@ -2393,56 +2531,57 @@ The CAA contactphone property takes a phone number as its parameter. The entire
-
-        applicantSigningNonce ATTRIBUTE ::= {
-            WITH SYNTAX              OCTET STRING
-            EQUALITY MATCHING RULE   octetStringMatch
-            SINGLE VALUE             TRUE
-            ID                       { cabf-applicantSigningNonce }
-        }
-
-        cabf-applicantSigningNonce OBJECT IDENTIFIER ::= { cabf 42 }
-        ```
-
-        The Random Value SHALL remain valid for use in a confirming response for no more than 30 days from its creation. The CPS MAY specify a shorter validity period for Random Values.
-        The CA MAY include a wildcard character in the Subject Alternative Name Extension and Subject Common Name Field as the left-most character in the .onion Domain Name provided inclusion of the wildcard character complies with [Section 3.2.2.6](#3226-wildcard-domain-validation) of these Requirements.
+   a. The CA MAY verify the Applicant’s control over the .onion service by using method in [Section 3.2.2.4.6 - Agreed‐Upon Change to Website](#32246-agreed-upon-change-to-website). If this method is replaced by a newer version(s) of Agreed-Upon Change to Website, the timelines for use of new and existing version of this method that are defined in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) SHALL apply.
+   b. The CA MAY verify the Applicant's control over the .onion service by having the Applicant provide a Certificate Request signed using the .onion public key if the Attributes section of the certificationRequestInfo contains:
+      i. A caSigningNonce attribute that contains a Random Value that is generated by the CA; and
+      ii. An applicantSigningNonce attribute that contains a single value with at least 64-bits of entropy that is generated by the Applicant.
+
+      The signing nonce attributes have the following format:
+
+      ```
+      caSigningNonce ATTRIBUTE ::= {
+         WITH SYNTAX              OCTET STRING
+         EQUALITY MATCHING RULE   octetStringMatch
+         SINGLE VALUE             TRUE
+         ID                       { cabf-caSigningNonce }
+      }
+
+      cabf-caSigningNonce OBJECT IDENTIFIER ::= { cabf 41 }
+
+      applicantSigningNonce ATTRIBUTE ::= {
+         WITH SYNTAX              OCTET STRING
+         EQUALITY MATCHING RULE   octetStringMatch
+         SINGLE VALUE             TRUE
+         ID                       { cabf-applicantSigningNonce }
+      }
+
+      cabf-applicantSigningNonce OBJECT IDENTIFIER ::= { cabf 42 }
+      ```
+
+      The Random Value SHALL remain valid for use in a confirming response for no more than 30 days from its creation. The CPS MAY specify a shorter validity period for Random Values.
+      The CA MAY include a wildcard character in the Subject Alternative Name Extension and Subject Common Name Field as the left-most character in the .onion Domain Name provided inclusion of the wildcard character complies with [Section 3.2.2.6](#3226-wildcard-domain-validation) of these Requirements.
```

Comment:
```suggestion
      The Random Value SHALL remain valid for use in a confirming response for no more than 30 days from its creation. The CPS MAY specify a shorter validity period for Random Values.

      The CA MAY include a wildcard character in the Subject Alternative Name Extension and Subject Common Name Field as the left-most character in the .onion Domain Name provided inclusion of the wildcard character complies with [Section 3.2.2.6](#3226-wildcard-domain-validation) of these Requirements.
```

Looks like the existing code was bugged here as well, in that the single line doesn't force a new paragraph.